akka-actor-tests/src/test/scala/akka/pattern/BackoffOnRestartSupervisorSpec.scala

```diff
@@ -52,7 +52,7 @@ class TestParentActor(probe: ActorRef, supervisorProps: Props) extends Actor {
 class BackoffOnRestartSupervisorSpec extends AkkaSpec with ImplicitSender {
 
   def supervisorProps(probeRef: ActorRef) = {
-    val options = Backoff.onFailure(TestActor.props(probeRef), "someChildName", 200 millis, 10 seconds, 0.0)
+    val options = Backoff.onFailure(TestActor.props(probeRef), "someChildName", 200 millis, 10 seconds, 0.0, -1)
```

Comment:
Even if private/test, perhaps consider using a named parameter here (`maxRestartAttempts = -1`) to make it easier to read.